### PR TITLE
MIA PT Miguel Maldonado

### PR DESCRIPTION
**Pull request by migmaldonado**
Thanks

### CODE REVIEW

File: starter-code/src/array-functions.js [hunk 1]
```diff
@@ -1,33 +1,38 @@
-var ArrayFunctions = function (){}
+
```
Comment (nickborbe) [inline starter-code/src/array-functions.js:1]: Good use of class syntax.  Why are you using static functions?

File: starter-code/src/array-functions.js [hunk 2]
```diff
@@ -1,33 +1,38 @@
...
 // 2) Define a function that takes an array with numbers and returns another array where each element contains double each element in the array
 // ArrayFunctions.doubleMyArray([10, 20, 35, 12])
 // [20, 40, 70, 24]
 
-ArrayFunctions.doubleArray = function(array){
-  return array.map(function(number){
-    return number * 2;
-  });
-};
+ArrayFunctions.doubleArray = (array => array.map(number => number * 2));
```
Comment (nickborbe) [inline starter-code/src/array-functions.js:27]: You already did this one up there ^

File: starter-code/src/array-functions.js [hunk 3]
```diff
@@ -1,33 +1,38 @@
...
+}
 
 // 1) Define a function that takes an array with numbers and prints all the elements of the array, separated by " --- "
 // ArrayFunctions.printSpecial([12, 33, 144, 122])
 // 12 -- 33 -- 144 -- 122
 
-ArrayFunctions.printSpecial = function (array) {
-  return array.join(" --- ");
-};
+ArrayFunctions.printSpecial = (array => array.join(" --- "));
```
Comment (nickborbe) [inline starter-code/src/array-functions.js:21]: This one too

File: starter-code/src/array-functions.js [hunk 4]
```diff
@@ -1,33 +1,38 @@
...
 // 54321
 // explanation: (1 x 10^0) + (2 x 10^1) + (3 x 10^2) + (4 x 10^3) + (5 x 10^4)
 // explanation: (1)        + (20)       + (300)      + (4000)     + (50000)
 
-ArrayFunctions.superPower = function(array){
-  return array.reduce(function(sum, number, index){
-    return sum + (number * (Math.pow(10, index)));
-  });
-}
+ArrayFunctions.superPower = array => array.reduce((sum, number, index) => sum + (number * (Math.pow(10, index))));
```
Comment (nickborbe) [inline starter-code/src/array-functions.js:35]: and this one

**Comment by nickborbe** [comment]
Good job.  You did some repetitive things but it's working.

File: starter-code/src/array-functions.js (hunk 2 above)
Comment (migmaldonado) [reply starter-code/src/array-functions.js:27]: I thought I had deleted or commented out the verbose syntax... whoops

**Pull request closed without merging**